Difi.SikkerDigitalPost.Klient.Domene/Extensions/EnumExtensions.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System;
+using Difi.Felles.Utility;
+
+namespace Difi.SikkerDigitalPost.Klient.Domene.Extensions
+{
+    internal static class EnumExtensions
+    {
+        public static string ToNorwegianString(this CertificateValidationType certificateValidationType)
+        {
+            switch (certificateValidationType)
+            {
+                case CertificateValidationType.Valid:
+                    return "Gyldig";
+                case CertificateValidationType.InvalidCertificate:
+                    return "UgyldigSertifikat";
+                case CertificateValidationType.InvalidChain:
+                    return "UgyldigKjede";
+                default:
+                    throw new ArgumentOutOfRangeException();
```

Comment:
Hva med å sende med en feilmelding med hvilket parameter som forårsaket exceptionet her? Ofte lettere å debugge hvis man får i en logg e.l. _hvilken_ `CertificateValidationType` som var ukjent.